game-authorization-score/src/main/java/com/iconbet/score/authorization/Authorization.java

```diff
@@ -0,0 +1,965 @@
+package com.iconbet.score.authorization;
+
+import com.eclipsesource.json.Json;
+import com.eclipsesource.json.JsonArray;
+import com.eclipsesource.json.JsonObject;
+import com.eclipsesource.json.JsonObject.Member;
+import com.eclipsesource.json.JsonValue;
+
+import java.math.BigInteger;
+import java.util.Arrays;
+import java.util.List;
+
+import score.Address;
+import score.ArrayDB;
+import score.Context;
+import score.DictDB;
+import score.VarDB;
+import score.annotation.EventLog;
+import score.annotation.External;
+import score.annotation.Payable;
+
+import com.iconloop.score.token.irc2.IRC2;
+
+
+public class Authorization implements IRC2{
```

Comment:
this is not a Token, this is just a contract, do not implement IRC2
The class in py does not inherits from a Token

class Authorization(IconScoreBase):

TokenStandard = IRC2.class

Reply:
IRC2 interface removed and methods as well